utils/utils.go

```diff
@@ -14,3 +18,29 @@ func RandStringBytes(n int) (string, error) {
 	}
 	return base64.StdEncoding.EncodeToString(b), nil
 }
+
+func GetInstancesInSecurityGroup(ctx context.Context, client *egoscale.Client, securityGroupID, zone string) ([]*v2.Instance, error) {
+	allInstances, err := client.ListInstances(ctx, zone)
+	if err != nil {
+		return nil, err
+	}
+
+	var instancesInSG []*v2.Instance
+	for _, instance := range allInstances {
+		for _, sgID := range *instance.SecurityGroupIDs {
+			if sgID == securityGroupID {
+				instancesInSG = append(instancesInSG, instance)
+			}
+		}
+	}
+
+	return instancesInSG, nil
+}
+
+func GetDefaultStringOrZero(v *string, defaultStr string) string {
```

Comment:
Same helper exists here:
https://github.com/exoscale/cli/blob/7ced7f01c89496050a7dc00c80a171c23a775fad/cmd/util.go#L34
I agree package `utils` is a better place for it though.